app/build.gradle

```diff
@@ -401,12 +401,13 @@ jar {
     from fileTree(buildDir).matching { include 'META-INF/services/*' }
 }
 
-task collectEnterpriseModules(dependsOn: [':users:jar']) {
+task collectEnterpriseModules(dependsOn: [':users:jar', ':ssl-impl:jar']) {
```

Comment:
There was a previous change to refactor the ssl handling (i.e. rejecting the psql sslrequest message) into a separate module `ssl`: 55f90207c71ecf2392e2067a19102a2ceed03bcf

This was necessary to decouple `ssl-impl` from `sql`. We decided in the planning that we don't want `ssl-impl` to depend on `sql` or the other way around.

The dependencies:
- `ssl-impl` depends on `ssl`
- `sql` depends on `ssl` but not on `ssl-impl` which enables us to plug in ssl support